packages/core/src/builtin-tasks/node.ts

```diff
@@ -0,0 +1,47 @@
+import { task, subtask } from '../config/config-env'
+import { paramTypes } from '../params'
+import { CkbNetManager } from '@kuai/docker-node'
+import '../type/runtime'
+
+interface Args {
+  port: number
+  detached: boolean
+}
+
+const TERMINATION_SIGNALS = ['SIGINT', 'SIGTERM', 'SIGQUIT']
+
+subtask('node:start', 'start a ckb node')
+  .addParam('port', 'The port of the node', 8114, paramTypes.number)
+  .addParam('detached', 'Run the node in detached mode', false, paramTypes.boolean)
+  .setAction(async ({ port, detached }: Args, env) => {
+    if (env.config.kuaiArguments?.network === 'docker-node') {
+      const ckbNetManager = await CkbNetManager.create()
+      ckbNetManager.start({ port: port.toString(), detached })
+
+      env.config.network = {
+        hostname: 'localhost',
+        port,
+      }
+    }
```

Comment:
Would it be better to throw an exception about using an unknown `network` instead of doing nothing in the else branch?